### ISSUE TITLE
Disable for Autoload Bale Loaders

### ISSUE THREAD

**Issue by BennyT93** [modelleicher/FS22_realismAddon_fillables#21]
Hi, my game crashed with the below lua error spammed in the log file when i overloaded passed 100% on an autoload bale loader. Might be best to disable this script for bale loaders if possible?

2024-05-21 21:01 Error: Running LUA method 'update'.
dataS/scripts/vehicles/specializations/BaleLoader.lua(2346) : attempt to index local 'balePlace' (a nil value)
[log_2024-05-21_19-58-37.txt](https://github.com/modelleicher/FS22_realismAddon_fillables/files/15395026/log_2024-05-21_19-58-37.txt)

**Comment by modelleicher** [comment]
Yeah I should do automatic disabling for bale loaders..  for now just disable overloading manually on automatic bale loaders
